hooks/shapediver/parameters/useParameterImportExport.ts

```diff
@@ -0,0 +1,162 @@
+import {NotificationContext} from "@AppBuilderShared/context/NotificationContext";
+import {useShapeDiverStoreParameters} from "@AppBuilderShared/store/useShapeDiverStoreParameters";
+import {useCallback, useContext} from "react";
+import {useShallow} from "zustand/react/shallow";
+
+/**
+ * Hook for managing parameter import/export and reset functionality.
+ */
+export function useParameterImportExport(namespace: string) {
+	const {batchParameterValueUpdate} = useShapeDiverStoreParameters(
+		useShallow((state) => ({
+			batchParameterValueUpdate: state.batchParameterValueUpdate,
+		})),
+	);
+
+	const notifications = useContext(NotificationContext);
+
+	/**
+	 * Export parameters as JSON file
+	 */
+	const exportParameters = useCallback(async () => {
+		try {
+			// Get current parameter values from store
+			const parameters = useShapeDiverStoreParameters
+				.getState()
+				.getParameters(namespace);
+			const parameterValues: {[key: string]: any} = {};
+
+			Object.values(parameters).forEach((paramStore) => {
+				const param = paramStore.getState();
+				parameterValues[param.definition.id] = param.state.uiValue;
+			});
```

Comment:
@LemToUp @mathieu-shapediver a question about the "export parameters as JSON" button: We disable this button if there are pending parameter changes, correct? We should not use `param.state.uiValue` here, because it's the value shown in the UI, which can be different from the "executed value"  `execValue`.

Reply:
@snabela Do you remember, is the `model.id` accessible at the app ?

Reply:
@LemToUp yes you can use `useShapeDiverStorePlatform.currentModel`. However, it might be undefined (e.g. if no slug was used). In that case we simply don't include the model id in the JSON.

Reply:
Done: b7f7b4a459c953728a609dd71d8e03f8f56a2edf